relay-server/src/actors/store.rs

```diff
@@ -1030,14 +988,28 @@ impl KafkaMessage {
     }
 }
 
-/// Message sent to the StoreForwarder containing an event
+/// Message sent to the [`StoreForwarder`] containing an event.
```

Comment:
```suggestion
/// Message sent to the [`StoreForwarder`] containing an [`Envelope`].
```

If we're being precise, we should rather describe what this message _does_ than what the struct is. This message produces items of an envelope on the appropriate kafka topics.